apps/coscrad-frontend/src/components/commands/command-panel.tsx

```diff
@@ -62,7 +62,14 @@ export const CommandPanel = ({ actions, commandContext }: CommandPanelProps) =>
     return (
         <CommandWorkspace
             selectedCommand={selectedCommand}
-            onFieldUpdate={updateForm}
+            onFieldUpdate={(p: string, v: unknown) => {
```

Comment:
Just wondering about `p` and `v`, are these temporary names?

Reply:
Yes- they are used for a log. You're right that I should remove this log!

Reply:
done